### PR TITLE
✨💥Add LuckyOne Animation

### PR DESCRIPTION
**Pull request by Sina-Sche**
Add Animation for result of the real decide by using dencrypt-effect (https://github.com/crazko/use-dencrypt-effect)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,16 @@
...
     "@tippyjs/react": "^4.2.0",
+    "bounty": "^1.3.0",
     "react": "^17.0.1",
+    "react-confetti": "^6.0.0",
     "react-dom": "^17.0.1",
+    "react-dom-confetti": "^0.2.0",
+    "react-rewards": "^1.1.2",
     "react-scripts": "4.0.0",
+    "react-spring": "^8.0.27",
+    "styled-components": "^5.2.1",
```
Comment (lmachens) [inline package.json]: I guess these dependencies are not required?

File: src/App.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import React from "react";
 import { useEffect, useState } from "react";
```
Comment (lmachens) [inline src/App.js]:
```suggestion
import React, { useEffect, useState } from "react";
```
or

```suggestion
import { useEffect, useState } from "react";
```

File: src/App.js [hunk 3]
```diff
@@ -31,7 +33,11 @@ function App() {
   const [round, setRound] = useState(
     JSON.parse(localStorage.getItem("round")) || 1
   );
+  const [show, setShow] = useState(false);
   const theLuckyHacker = "Benjamin Weinstock";
+
+  const handleShow = () => {};
```
Comment (lmachens) [inline src/App.js]:
```suggestion
```

File: src/App.js [hunk 4]
```diff
@@ -58,6 +64,9 @@ function App() {
     <div className="container">
       <Header />
       <main className="main">
+        <div>
+          <Confettis recycle={show} />
+        </div>
```
Comment (lmachens) [inline src/App.js]: I guess Confettis is an old test and can be removed?

File: src/components/Animation.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```
Comment (lmachens) [inline src/components/Animation.js]:
```suggestion
import { useState } from "react";
```

File: src/components/Animation.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        i = i === values.length - 1 ? 0 : i + 1;
+        if (i === 0) {
+          setFinished(true);
+          clearInterval(action);
+        }
+      }, 1000);
+
+      return () => clearInterval(action);
+    }
+  });
```
Comment (lmachens) [inline src/components/Animation.js]:
```suggestion
  }, [finished]);
```

File: src/components/TheLuckyOne.js [hunk 7]
```diff
@@ -1,11 +1,35 @@
 import "./TheLuckyOne.css";
```
Comment (lmachens) [inline src/components/TheLuckyOne.js]: Duplicate of Animations.js :)

**Review by lmachens — COMMENTED** [review]
gj 🎊

File: package.json (hunk 1 above)
Comment (Sina-Sche) [reply package.json]: Done 💯

File: src/components/TheLuckyOne.js (hunk 7 above)
Comment (Sina-Sche) [reply src/components/TheLuckyOne.js]: Thanks for your review! 👍

**Review by bwnstck — APPROVED** [review]
subber🚴‍♀️

**Pull request merged**